view/templates/htconfig.tpl

```diff
@@ -1,5 +1,12 @@
 <?php
 
+/**********************************************************************
```

Comment:
can you write `/* **********` instead of `/**********` because otherwise it would be recognized as oxygen docu